curator-recipes/src/main/java/org/apache/curator/framework/recipes/locks/InterProcessReadWriteLock.java

```diff
@@ -114,17 +114,7 @@ public static class WriteLock extends InternalInterProcessMutex
     {
         public WriteLock(CuratorFramework client, String basePath, byte[] lockData)
         {
```

Comment:
This old code doesn't do anything right?

Reply:
Yep, I have same thought.